asconfig.json

```diff
@@ -14,7 +14,12 @@
       "shrinkLevel": 1,
       "converge": false,
       "noAssert": false
+    },
+    "test": {
+      "debug": true
     }
   },
-  "options": {}
+  "options": {
+    "transform": ["visitor-as/dist/examples/includeBytesTransform"]
```

Comment:
Just wondering what this transform does and why it is needed?

Reply:
It allows you to embed the bytes of a file into a static array. Akin to the rust function `include_bytes`.